include/deal.II/grid/grid_tools.h

```diff
@@ -931,29 +931,25 @@ namespace GridTools
     /**
      * The relative orientation of the first face with respect to the second
      * face as described in orthogonal_equality() and
-     * make_periodicity_constraints() (and stored as a bitset).
+     * DoFTools::make_periodicity_constraints() (and stored as a bitset).
      */
     std::bitset<3> orientation;
 
     /**
-     * A matrix that describes how vector valued DoFs of the first face should
-     * be modified prior to constraining to the DoFs of the second face. If
-     * the std::vector first_vector_components is non empty the matrix is
-     * interpreted as a @p dim $\times$ @p dim rotation matrix that is applied
-     * to all vector valued blocks listed in @p first_vector_components of the
-     * FESystem. If @p first_vector_components is empty the matrix is
-     * interpreted as an interpolation matrix with size no_face_dofs $\times$
-     * no_face_dofs. For more details see make_periodicity_constraints() and
+     * A matrix that describes how vector valued DoFs of the first face
+     * should be modified prior to constraining to the DoFs of the second
+     * face. If the std::vector first_vector_components (supplied as a
+     * parameter to DofTools::make_periodicity_constraints()) is non empty
+     * the matrix is interpreted as a @p dim $\times$ @p dim rotation
+     * matrix that is applied to all vector valued blocks listed in @p
+     * first_vector_components of the FESystem. Alternatively, if @p
```

Comment:
Well, the vector has not been removed completely, but just in GridTools::collect_periodic_faces. The meaning of this matrix depends on its size. In case of a dim x dim rotation matrix, we have to specify in DoFTools::make_periodicity_constraints on which vector-valued components we want to apply it .